codalab/worker/main.py

```diff
@@ -33,7 +33,9 @@
 def parse_args():
     parser = argparse.ArgumentParser(description='CodaLab worker.')
     parser.add_argument(
-        '--tag', help='Alphanumeric tag that allows for scheduling runs on specific workers.'
+        '--tags',
```

Comment:
The one thing I'm not comfortable about is that this is a breaking change (passing --tags instead of --tag), so this would probably necessitate a major version bump. Maybe would it be less disruptive just to allow people to pass in `--tag` instead?

Reply:
Yeah, I would prefer to support `--tag` and just let the user pass in multiple tags in that argument if needed. We could support both if you think it's better, though, not super opinionated on that point.

Reply:
Hey @nelson-liu would you like to / still have time to make this change? renaming it to `--tag`?